From 94dec7ffa2bedda1c00c863879d44a6d35e9a9de Mon Sep 17 00:00:00 2001
From: Daniel Dragomir <daniel.dragomir@windriver.com>
Date: Tue, 8 Aug 2017 19:16:59 +0300
Subject: [PATCH 243/278] tools/perf: Correct the hexa value 0x1ULL from
 opencsd

Fix a typo from the porting of perf cs-etm decoder for OpenCSD from
the following repo. Only the hexa prefix and the unsigned long long
suffix are present with no value.

repository: https://github.com/Linaro/OpenCSD.git
branch: perf-opencsd-4.9
commit e38cbdb2928a36c5a ("cs-etm: Update to perf cs-etm decoder for OpenCSD v0.5")

Signed-off-by: Daniel Dragomir <daniel.dragomir@windriver.com>
Signed-off-by: John Jacques <john.jacques@intel.com>
---
 tools/perf/util/cs-etm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/perf/util/cs-etm.c b/tools/perf/util/cs-etm.c
index 91d6a8a..5fcd155 100644
--- a/tools/perf/util/cs-etm.c
+++ b/tools/perf/util/cs-etm.c
@@ -1190,7 +1190,7 @@ static int cs_etm__process_event(struct perf_session *session,
 					etmq->decoder,
 					foffset,
 					fstart,
-					fsize & ~0xULL,
+					fsize & ~0x1ULL,
 					symbol_conf.vmlinux_name);
 
 				etmq->kernel_mapped = true;
-- 
1.8.5.2.233.g932f7e4

